inc/osdconfig.h

```diff
@@ -7,10 +7,10 @@
 #define EERROM_SIZE                             1024
 
 // Version number: major.minor.revision (1.2.6 for example)
-#define PLAYUAV_VERSION_NUMBER          "1.1.2"
+#define PLAYUAV_VERSION_NUMBER          "1.1.3"
```

Comment:
Sure, no problem. I'm not fighting you on the semver stuff, simply unfamiliar. As long as the numbers go up I'm happy.